OK, I can see:

```
2017-05-10 16:00:13,933 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: ERROR

2017-05-10 16:00:13,936 5784 INFO openerp_test odoo.addons.sale_validity.tests.test_sale_validity: ======================================================================

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: ERROR: test_sale_validity (odoo.addons.sale_validity.tests.test_sale_validity.TestSaleValidity)

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: Traceback (most recent call last):

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/build/OCA/sale-workflow/sale_validity/tests/test_sale_validity.py", line 13, in test_sale_validity

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     so_no_validity = self.create_so()

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/build/OCA/sale-workflow/sale_validity/tests/test_sale_validity.py", line 27, in create_so

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     so = self.env['sale.order'].create(vals)

2017-05-10 16:00:13,937 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/OCB-10.0/addons/sale/models/sale.py", line 271, in create

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     result = super(SaleOrder, self).create(vals)

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/OCB-10.0/addons/mail/models/mail_thread.py", line 228, in create

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     thread = super(MailThread, self).create(values)

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/OCB-10.0/odoo/models.py", line 3830, in create

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     record = self.browse(self._create(old_vals))

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/OCB-10.0/odoo/models.py", line 3987, in _create

2017-05-10 16:00:13,938 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     field.write(self.with_context(rel_context), vals[name])

2017-05-10 16:00:13,939 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/OCB-10.0/odoo/fields.py", line 2174, in write

2017-05-10 16:00:13,939 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     comodel.create(act[2])

2017-05-10 16:00:13,939 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `   File "/home/travis/OCB-10.0/addons/sale_margin/models/sale_order.py", line 50, in create

2017-05-10 16:00:13,939 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: `     product_uom_id = self.env['product.uom'].browse(vals['product_uom'])

2017-05-10 16:00:13,939 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: ` KeyError: 'product_uom'

2017-05-10 16:00:13,939 5784 INFO openerp_test odoo.addons.sale_validity.tests.test_sale_validity: Ran 1 test in 0.088s

2017-05-10 16:00:13,939 5784 ERROR openerp_test odoo.addons.sale_validity.tests.test_sale_validity: FAILED

2017-05-10 16:00:13,939 5784 INFO openerp_test odoo.addons.sale_validity.tests.test_sale_validity:  (errors=1)

2017-05-10 16:00:13,940 5784 ERROR openerp_test odoo.modules.module: Module sale_validity: 0 failures, 1 errors

2017-05-10 16:00:13,952 5784 ERROR openerp_test odoo.modules.registry: Failed to load registry
```

--- a/sale_order_margin_percent/README.rst
+++ b/sale_order_margin_percent/README.rst
@@ -6,28 +6,7 @@
 Sale Order Margin Percent
 =========================
 
-This module show the margin profit in sales and invoicing.
-It can be used both in budgets and in sales orders,
-in the sales section and for invoicing can also be used.
-
-============
-Installation
-============
-
-To install this module, you need to:
-
-- First Add in the addons path the module and install in your Odoo, And install
-  the dependencies of the module.
-
-=============
-Configuration
-=============
-
-To configure this module, you need to:
-
-- First Add in the sales configuration the option
-  "Show margins in quotes and sales orders" then you can get the margin field
-  for the operations.
+This module shows the margin profit percentage in sales orders.
 
 =====
 Usage
@@ -35,17 +14,17 @@ Usage
 
 To use this module, you need to:
 
-#. Go to sales and enter in sale order section.
-#. create a sale order then add products.
-#. You can get the percent because you have values in the field margin and amount untaxed.
-#. For the invoicing is the same enter an create a new one.
+#. Go to sales and enter in sale order section,
+#. create a sale order then add products,
+#. You can get the percent because you have values in the field margin and
+amount untaxed.
 
 ===========
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/project_task_analytic_tag/issues>`_. In case of trouble,
+<https://github.com/OCA/sale-workflow/issues>`_. In case of trouble,
 please check there if your issue has already been reported. If you spotted it
 first, help us smash it by providing detailed and welcomed feedback.
 

